server/db/018-sethubinstallationidtonotnull.sql

```diff
@@ -0,0 +1,22 @@
+DO $migration$
+    DECLARE migrationId INT;
+    DECLARE lastSuccessfulMigrationId INT;
+BEGIN
+    -- The migration ID of this file
+    migrationId := 18;
+
+    -- Get the migration ID of the last file to be successfully run
+    SELECT MAX(id) INTO lastSuccessfulMigrationId
+    FROM migrations;
+
+    -- Only execute this script if its migration ID is next after the last successful migration ID
+    IF migrationId - lastSuccessfulMigrationId = 1 THEN
+        -- ADD SCRIPT HERE
+        ALTER TABLE hubs
+        ALTER COLUMN installation_id SET NOT NULL;
```

Comment:
Note to self: Do not merge until after #123 is DEPLOYED and `installation_id` has been given a value for all hubs.